guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,29 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {RHEL} 7 to {RHEL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
```

Comment:
@maximiliankolb Do you think it would be useful to generalize this procedure for EL systems?

Reply:
@ekohl Thanks!